flake.nix

```diff
@@ -264,6 +266,25 @@
           '';
         };
 
+        package-sets = {
+          flakeRef = "github:ConnorBaker/package-sets";
+          baseUrl = "https://github.com/ConnorBaker/package-sets";
+          intro = ''
+            Populate `packages` and `overlays` with ease.
+
+            [package-sets](https://github.com/ConnorBaker/package-sets) was born out of frustration with the manual
+            process of adding packages and overlays to a flake. If your flakes have more than a few packages,
+            dependencies between packages, or require Nixpkgs use a non-standard configuration, this module is for you.
+
+            ## Limitations
+            
+            A great deal of this module's functionality is based on `lib.packagesFromDirectoryRecursive`.
```

Comment:
Couldn't you pass the output of `packagesFromDirectoryRecursive` as an option default?